cmd/dptp-controller-manager/main.go

```diff
@@ -60,7 +61,9 @@ func (o *options) addDefaults() {
 }
 
 type testImagesDistributorOptions struct {
-	imagePullSecretPath string
+	imagePullSecretPath          string
+	whiteListedImageStramTagsRaw flagutil.Strings
+	whiteListedImageStreamTags   sets.String
```

Comment:
I don't insist on it, but I saw @thiagoalessio 's post on aos-devel about calling this with neutral `allowlist/denylist` terminology instead, and thought it's a good idea. WDYT? (I also want to address this in our other tools where we already have allowlists).

Reply:
Good point, I changed the whole thing to `additional-image-stream-tag`, the whole allow-list name was an artifact of how its implemented which shouldn't play a role in the flag naming